packages/react/src/contexts/index.tsx

```diff
@@ -0,0 +1,125 @@
+import { Children, createContext, Fragment, isValidElement, PropsWithChildren, ReactNode, useContext } from "react";
+import type { RoboWizard, StepConfig } from "robo-wizard";
+import { useWizard } from "../hooks";
+
+/**
+ * @param props - accepts same props to match [[StepConfig]] for [[useWizard]], as well as the `element` to display
+ **/
+export function Step(_props: StepConfig & { element: ReactNode | null }): React.ReactElement | null {
+  return null;
+}
+
+/**
+ * @param steps = The Children opaque object passed to components, only Step components will be processed
+ **/
+export function createStepsFromChildren(steps: ReactNode): Array<StepConfig & { element: ReactNode }> {
```

Comment:
Inspired by the [`createRoutesFromElements`](https://reactrouter.com/en/main/utils/create-routes-from-elements) function from React Router. This has potential reuse in the `@robo-wizard/react-router` package.